### PR TITLE
Update new SSO config instructions for generic SAML, Okta, Gsuite and Azure AD

### PR DESCRIPTION
**Pull request by ajaythomas**
## What are you changing in this pull request and why?
Updated new SSO config instructions for 
- generic SAML
- Okta
- GSuite
- Azure AD

Added screenshots, moved table containing auth0_uri to main SSO page.

https://dbtlabs.atlassian.net/browse/ENTERPRISE-529

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [x] Added new screenshots for SSO configs
- [x] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [x] Rename file names of older screenshots

### CODE REVIEW

File: website/docs/docs/cloud/about-cloud/regions-ip-addresses.md [hunk 1]
```diff
@@ -9,12 +9,12 @@ dbt Cloud is [hosted](/docs/cloud/about-cloud/architecture) in multiple regions
...
-| APAC  [^1] | 	AWS ap-southeast-2  (Sydney)| au.dbt.com | us-production-mt.us.auth0.com | urn:auth0:us-production-mt | 52.65.89.235 <br /> 3.106.40.33 <br /> 13.239.155.206 <br />| ❌ | ❌ | ✅ |
-| Virtual Private dbt or Single tenant | Customized |  Customized | Ask [Support](/guides/legacy/getting-help#dbt-cloud-support) for your Auth IP | Ask [Support](/guides/legacy/getting-help#dbt-cloud-support) for your Audience URN | Ask [Support](/guides/legacy/getting-help#dbt-cloud-support) for your IPs | ❌ | ❌ | ✅ |
+| North America [^1] | AWS us-east-1 (N. Virginia) | cloud.getdbt.com | 52.45.144.63 <br /> 54.81.134.249 <br />52.22.161.231 | ✅ | ✅ | ✅ |
+| EMEA [^1] | AWS eu-central-1	(Frankfurt) | emea.dbt.com | 3.123.45.39 <br /> 3.126.140.248 <br /> 3.72.153.148 | ❌ | ❌ | ✅ |
+| APAC  [^1] | 	AWS ap-southeast-2  (Sydney)| au.dbt.com | 52.65.89.235 <br /> 3.106.40.33 <br /> 13.239.155.206 <br />| ❌ | ❌ | ✅ |
+| Virtual Private dbt or Single tenant | Customized |  Customized | Ask [Support](/guides/legacy/getting-help#dbt-cloud-support) for your IPs | ❌ | ❌ | ✅ |
```
Comment (ajaythomas) [inline website/docs/docs/cloud/about-cloud/regions-ip-addresses.md:17]: It was confusing to add the auth0 uri and SAML specific values in this table. Moved it to `website/docs/docs/cloud/manage-access/sso-overview.md`

File: website/snippets/access_url.md [hunk 2]
```diff
@@ -1 +1 @@
-The following steps use `YOUR_AUTH_URL` and `YOUR_AUDIENCE_URN`, which need to be replaced with the [appropriate Auth URL and Audience URN](docs/cloud/about-cloud/regions-ip-addresses) for your region and plan.
+The following steps use `YOUR_AUTH0_URI` and `YOUR_AUTH0_ENTITYID`, which need to be replaced with the [appropriate Auth0 SSO URI and Auth0 Entity ID](docs/cloud/manage-access/set-up-sso-saml-2.0#auth0-multi-tenant-uris) for your region.
```
Comment (ajaythomas) [inline website/snippets/access_url.md]: I am referencing this link that the deploy did not like. How do I build this link @matthewshaver @aaronbini

Comment (ajaythomas) [reply website/snippets/access_url.md]: It should come from [here](https://github.com/dbt-labs/docs.getdbt.com/pull/3516/files#diff-7471259c186c98aa73c3f6c60767b8f85e50a7d16308a0873a6488ce9420c6e0R22)

Comment (matthewshaver) [inline website/snippets/access_url.md]:
```suggestion
The following steps use `YOUR_AUTH0_URI` and `YOUR_AUTH0_ENTITYID`, which need to be replaced with the [appropriate Auth0 SSO URI and Auth0 Entity ID](/docs/cloud/manage-access/set-up-sso-saml-2.0#auth0-multi-tenant-uris) for your region.
```

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**